### DISCUSSION TITLE
wix4: HarvestDirectory and LocalAppDataFolder

### DISCUSSION THREAD

**Discussion by micktonnelier** [wixtoolset/issues#7938] [Questions]
Hello,

I do not able to use HarvestDirectory with LocalAppDataFolder installation.
For my project, I used HarvestDirectory for retrieve all assets files (and sub-assets file) that were generated automatically.
I don' t have any problem, if my target installation is on `ProgramFiles6432Folder`. Everythink works fine (upgrade and uninstall too).

But now, I want to be able to install my app without admin privilege, so I changed the target to `LocalAppDataFolder` and now...
I have +400 errors like this:
`error WIX0204: ICE64: The directory dirqk5NfPX27WaAgtTfBL6WgpZ9x4I is in the user profile but is not listed in the RemoveFile table.` 
or like that:
`error WIX0204: ICE38: Component cmpyy05vabdgfiy9dwGgGBnBY.fdCw installs to user profile. It must use a registry key under HKCU as its KeyPath, not a file.`
The name was generated by HarvestDirectory, that ok. But I tried many combination of RemoveFolder/RemoveFile but none of them works.

Does somebody can help me ?

**Comment by micktonnelier** [comment]
Currently, It's work with that, but I'd prefer another solution. 

```
<SuppressIces>ICE64;ICE38;ICE91</SuppressIces>
```

**Reply by rainbowechoes** [reply]
yes, this way can work. I'm also searching for another way to resolve it.

I try to change the xslt file to add RegistryKey tag under every File tag. but it requires me can't use Guid='*'. and I don't know how to set a guid value for the Guid attribute by myself.

here is my question link:
https://github.com/orgs/wixtoolset/discussions/7940

**Comment by robmen** [comment]
Those are ICE errors. The ICEs come from the Windows Installer team. They provide some additional documentation, like [ICE64](https://learn.microsoft.com/en-us/windows/win32/msi/ice64).
